README.md

```diff
@@ -39,7 +39,7 @@ If you want to update a LoRa Gateway running our software 0.2.0-preview to the c
 
 ## Current limitations
 
-- Multigateway works but is not fully tested and you need to implement message deduplication after IoT Hub, if multiples gateways are used in the same range of the device we recommend setting the gateway tag "GatewayID" on the device twins with the IoT Edge ID of the preferred gateway for that device.
+- We support multigateway but currently you need to implement message deduplication after IoT Hub, if multiples gateways are used in the same range of the device and you don't want duplicate messages or redundancy we recommend setting the gateway tag "GatewayID" on the device twins with the IoT Edge ID of the preferred gateway for that device.
```

Comment:
Do you want to mention about Stream Analytics as a possible solution?

Reply:
Without testing it is just a educated guess. View that Patrick is already working on dedup I don't want to mention ASA yet